commerce-theme-minium/commerce-theme-minium/src/css/minium/_mixins.scss

```diff
@@ -1,32 +1,40 @@
-@mixin minium-typo-title {
-	color: $color-fg;
+@mixin typo-text {
 	font-size: 14px;
+	line-height: 1.6;
+}
+
+@mixin base-title {
 	font-weight: 500;
-	letter-spacing: 3px;
-	line-height: 19px;
-	margin: $minium-big-gutter 0;
+	letter-spacing: 1.2px;
+	line-height: 1;
 	text-transform: uppercase;
 }
 
-@mixin minium-typo-text {
+@mixin minium-typo-title {
+	@include base-title;
+
+	color: $color-fg;
 	font-size: 14px;
-	line-height: 32px;
+	margin: $lg-gutter 0;
 }
 
-@mixin minium-panel-title {
+@mixin card-title {
+	@include base-title;
+
+	font-size: 12px;
```

Comment:
looks like this is the only unsorted rule in this file, as far as I can tell -- EDIT: oh dang, you already fixed it! Thanks :). Ignore.